src/FINDOLOGIC/Export/Helpers/UsergroupAwareMultiValueItem.php

```diff
@@ -46,7 +51,7 @@ public function getDomSubtree(\DOMDocument $document)
     /**
      * @inheritdoc
      */
-    public function getCsvFragment()
+    public function getCsvFragment(array $availableProperties = [])
     {
         return $this->value;
```

Comment:
use the `getValue()` method in the whole class as you already implemented it

Reply:
:heavy_check_mark: